wpt/tests/browsing_context/event_context_created.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_browsing_context_context_created_emitted(bidi_session,
+      wait_for_event, recursive_compare):
+
+    await bidi_session.session.subscribe(
+        events=["browsingContext.contextCreated"])
+
+    on_context_created_promise = wait_for_event("browsingContext.contextCreated")
+
+    create_command_promise = await bidi_session.send_command(
+        "browsingContext.create", {})
+
+    context_created_event = await on_context_created_promise
+
+    recursive_compare({
+        "context": "__any_value__",
```

Comment:
In lots of places where "__any_value__" the tests would actually pass if the property isn't there at all, due to how `recursive_compare` works. Do you think this is a risk? Should tests fail if the attribute isn't there at all?

Reply:
We check if the keys set in expected and actual objects are equal: [`assert expected.keys() == actual.keys()`](https://github.com/GoogleChromeLabs/chromium-bidi/blob/16031b49b6c474c1dc4400a35e93959ee6212ac8/wpt/fixtures.py#L92).